checkov/dockerfile/checks/RunUsingAPT.py

```diff
@@ -0,0 +1,25 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.dockerfile.base_dockerfile_check import BaseDockerfileCheck
+
+
+class RunUsingAPT(BaseDockerfileCheck):
+    def __init__(self):
+        """
+        Apt interface is less stable than apt-get and so this preferred
+        """
+        name = "Ensure that APT isn't used"
+        id = "CKV_DOCKER_9"
+        supported_instructions = ["*"]
```

Comment:
```suggestion
        supported_instructions = ["RUN"]
```
then you shouldn't need to check later, if `instruction == "RUN"`

Reply:
spot on.